managed_vms/jetty9-gcloud-sessions/src/main/jetty-base/etc/jetty-gcloud-sessions.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <!-- ============================================================================================== -->
+  <!-- GCloud configuration.                                                                          -->
+  <!-- This replaces the file in jetty releases < 9.3.8 with a fix to extract the worker name from    -->
+  <!-- the GAE_MODULE_INSTANCE env variable.  This can be removed from jetty-9.3.8 onwards.           -->
+  <!-- ============================================================================================== -->
+  <New id="gconf" class="org.eclipse.jetty.gcloud.session.GCloudConfiguration">
+    <!-- To contact remote gclouddatastore set the following properties in start.ini -->
+    <!-- Either set jetty.gcloudSession.projectId or use system property/env var DATASTORE_DATASET-->
+    <Set name="projectId"><Property name="jetty.gcloudSession.projectId"/></Set>
+    <Set name="p12File"><Property name="jetty.gcloudSession.p12File"/></Set>
```

Comment:
@lesv Note sure I'm following you here as to what is ancient history?
The `jetty-gcloud-sessions.xml` is the current way we configure the gcloud session manager in the current jetty release.  The change in this PR was to add the Env lookup for `GAE_MODULE_INSTANCE`

So are you saying that the ancient history is the form of the gcloud data store configuration that jetty is currently using?    I think @janbartel has been having issues navigating exactly which are current APIs to use and what exactly is current best practise.  So if you have any pointers for "Usable Auth" and the JSON file we should be using to configure, then we will look to update the session manager to use them.

However, without such an update, does the ancient history of this approach negate the value of this example?  Ie should we complete this example with the current session manager or wait until we can update the session manager to a more recent version?

Perhaps we should do the example with the JDBC session manager instead?